pkg/inits/config.go

```diff
@@ -27,7 +27,7 @@ func GetConfig() *entities.Config {
 	once.Do(func() {
 		viper.SetConfigName("config")
 		viper.SetConfigType("yaml")
-		viper.AddConfigPath("../../")
+		viper.AddConfigPath("/root")
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2426343786>#df96d7</a><span> - 10/21/2024, 04:38 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The hardcoded configuration path '/root' may cause issues in non-Docker environments and limit flexibility.
</div>
<div id="fix">
<b>Fix:</b> Implement a more flexible configuration path strategy that works across different environments.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -30,1 +30,5 @@
 -		viper.AddConfigPath("/root")
 +		viper.AddConfigPath("/root") // Docker environment
 +		viper.AddConfigPath(".")     // Current directory
 +		viper.AddConfigPath("$HOME/.productivite") // Home directory
 +		viper.AddConfigPath("/etc/productivite")   // System-wide directory
 +		viper.AddConfigPath("../../") // Project root (for local development)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged